Source/Scene/Model.js

```diff
@@ -517,6 +521,24 @@ define([
         this._shadows = this.shadows;
 
         /**
+         * A color that blends with the model's rendered color.
+         *
+         * @type {Color}
+         *
+         * @default Color.RED
+         */
+        this.blendColor = defaultValue(options.blendColor, Color.RED);
```

Comment:
I had intentionally left alpha out, but I do think it will be nice to support in this PR.